### ISSUE TITLE
Crash when processing 0-hitobject beatmap

### ISSUE THREAD

**Issue by kiwec** [MaxOhn/rosu-pp#5]
```
thread 'main' panicked at 'attempt to subtract with overflow', /home/kiwec/.cargo/registry/src/github.com-1ecc6299db9ec823/rosu-pp-0.4.0/src/osu/mod.rs:287:28
```

The beatmap: https://osu.ppy.sh/beatmapsets/195741  
Title might be wrong since the website says it has 2 sliders, but idk.  
Alternative: https://osu.ppy.sh/beatmapsets/194899#osu/470113

https://github.com/MaxOhn/rosu-pp/blob/main/src/osu/mod.rs#L287

**Comment by kiwec** [comment]
Another one: https://osu.ppy.sh/beatmapsets/614120

**Comment by MaxOhn** [comment]
Ugh I accidentally re-introduced that issue with my most recent refactoring, thanks for letting me know.
Fixed with #6 but I'll likely wait a little until I put it onto `main` and release it since it's only an edge case and not a pressing issue.
Will close this issue once I fully merge it.

**Comment by MaxOhn** [comment]
New release is out, fix is merged 👌
